pkg/apis/compliance/v1alpha1/tailoredprofile_types.go

```diff
@@ -6,6 +6,21 @@ import (
 
 // FIXME: move name/rationale to a common struct with an interface?
 
+// DisableOutdatedReferenceValidation a label is used to disable validation of outdated references
+const DisableOutdatedReferenceValidation = "compliance.openshift.io/outdated-reference-validation-disable"
```

Comment:
Nit:
```suggestion
const DisableOutdatedReferenceValidation = "compliance.openshift.io/disable-outdated-reference-validation"
```

Reply:
What is the expected usage for this label?
It is not assigned by the profile parser, is it just for testing purposes?
Do we expect the user to use it somehow?

Reply:
these annotations are being used on tailoredProfile, the customer can use it maybe for advance use case when they want to suppress  the warnings.